testing/src/test/java/io/stargate/it/http/GraphqlTest.java

```diff
@@ -135,8 +135,9 @@
  *       see generated code in `target/generated-sources/graphql-client`.
  * </ul>
  */
-@NotThreadSafe
+@Isolated
 public class GraphqlTest extends BaseOsgiIntegrationTest {
```

Comment:
I kind of liked the transition to the standard `*IT` naming convention, but I see you've reverted that... At the end of the day it doesn't matter that much.

Reply:
Same. Wasn't sure what everyone's opinion of it was and since we weren't using failsafe for parallelism it wasn't really needed. We can switch it back later.

Reply:
Since we do not have any proper "unit" tests in the `testing` module we might as we use the `surefire` plugin to run the tests, thus removing the naming ambiguity.